tests/e2e/provisioning/test/e2e_provisioning_test.go

```diff
@@ -0,0 +1,25 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_E2E_Provisioning(t *testing.T) {
+	ts := newTestSuite(t)
+	operationID, err := ts.brokerClient.ProvisionRuntime()
+	require.NoError(t, err)
+	defer ts.TearDown()
+
+	err = ts.brokerClient.AwaitProvisioningSucceeded(operationID)
+	require.NoError(t, err)
+
+	dashboardURL, err := ts.brokerClient.FetchDashboardURL()
+	require.NoError(t, err)
+
+	err = ts.kymaClient.CallDashboardURL(dashboardURL)
```

Comment:
```suggestion
	err = ts.kymaClient.AssertRedirectedToUAA(dashboardURL)
```

define your expectation instead what is your action